.devcontainer/Dockerfile.backend

```diff
@@ -7,3 +7,6 @@ RUN apt-get update \
     sqlite3 pre-commit \
     libopencv-dev clang libclang-dev \
     && rm -rf /var/lib/apt/lists/*
+
+RUN curl -sSL https://rover.apollo.dev/nix/latest | sh \
+    && curl -sSL https://router.apollo.dev/download/nix/latest | sh
```

Comment:
Needs newline at EOF

Reply:
I've remove this, we can add it with the supergraph PR